config/config.go

```diff
@@ -657,6 +641,22 @@ func (c *MarathonSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) erro
 	return checkOverflow(c.XXX, "marathon_sd_config")
 }
 
+// KubernetesSDConfig is the configuration for Kubernetes service discovery.
+type KubernetesSDConfig struct {
+	Masters         []URL      `yaml:"masters"`
```

Comment:
Yeah this is a legacy thing. I'll change it. Do I need to deprecate `masters ` & handle both options for a release or OK to make the breaking change? Seeing as beta only right now I'd go for breaking change?